cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```

Comment:
To avoid this waring:
![image](https://user-images.githubusercontent.com/10209180/179674076-1677e58a-6506-4f86-9343-4070d6627c62.png)

You should add an empty line at the end of the file (or linebreak), that line is called EOF. You can configure your visual studio code to add automatically the EOF: https://stackoverflow.com/a/44704969 

Next time that we see this, we will comment just EOF as a reminder 😊